### PR TITLE
John conroy/home revision counts

### PR DESCRIPTION
**Pull request by john-conroy**
[Design](https://www.figma.com/file/cEZdI2tOMByTBW8RQ2kuAm/Home-Page?node-id=555%3A1). No cells yet. I'm also using MUI's [skeleton](https://material-ui.com/components/skeleton/) for the first time. Again, I'd check this out locally on different screen sizes and maybe use responsive to check resizing.

desktop/laptop

Screen Shot 2021-04-16 at 1 48 41 PM

desktop/laptop hover

Screen Shot 2021-04-16 at 2 04 04 PM

mobile

Screen Shot 2021-04-16 at 1 50 21 PM

This last layout is only visible on a narrow range of screen widths between mobile and desktop/laptop.

Screen Shot 2021-04-16 at 1 50 03 PM

### CODE REVIEW

**Comment by tsliaw** [comment]
On larger screens (i.e. a monitor), there is an awkward spacing between the carousel and the counts if you can fix that. I've attached a screenshot for your reference. 
Screen Shot 2021-04-19 at 9 38 45 AM

File: context/app/static/js/components/home-revision/EntityCounts/EntityCounts.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          label="Samples"
+          href="/search?entity_type[0]=Sample"
+        />
+        <EntityCount
+          icon={DatasetIcon}
+          count={entityCounts?.Dataset}
+          label="Datasets"
+          href="/search?entity_type[0]=Dataset"
+        />
+        <EntityCount icon={CollectionIcon} count={entityCounts?.Collection} label="Collections" href="/collections" />
```
Comment (ilan-gold) [inline context/app/static/js/components/home-revision/EntityCounts/EntityCounts.jsx:45]: Not sure if this is cleaner (take it or leave it) but you could make this into a `[].map` to make it less copy-and-paste-y:
```suggestion
        [
        [DonorIcon, entityCounts?.Donor, "Donors", "/search?entity_type[0]=Donor"],
       [SampleIcon,entityCounts?.Sample,"Samples","/search?entity_type[0]=Sample"],         
     [DatasetIcon,entityCounts?.Dataset,"Datasets","/search?entity_type[0]=Dataset"],
     [CollectionIcon, entityCounts?.Collection,"Collections","/collections"]
       ].map(([icon, count, label, href]) => (
       <EntityCount
          icon={icon}
          count={count}
          label={label}
          href={href}
        />
       ));
```

**Review by ilan-gold — COMMENTED** [review]
I couldn't test it but the code looked good.  Got a series of errors like:

```
17:26  error  Unable to resolve path to module 'portal-images/ccf-slide-320w.png'        import/no-unresolved
```

File: context/app/static/js/components/home-revision/EntityCounts/EntityCounts.jsx (hunk 1 above)
Comment (john-conroy) [reply context/app/static/js/components/home-revision/EntityCounts/EntityCounts.jsx:45]: Thanks, I'll do something similar.

**Comment by john-conroy** [comment]
@ilan-gold The errors probably have something to do with the new submodule. You might need to pull it down.

**Comment by john-conroy** [comment]
@tsliaw as we discussed I'll remove the above the fold approach, but in a future PR.

**Comment by ilan-gold** [comment]
Could you provide the specific command to pull?  I tried the `README.md` but had no luck.  When I look at https://github.com/hubmapconsortium/portal-ui/tree/john-conroy/home-revision-counts/context I don't see a submodule.

**Comment by john-conroy** [comment]
If you have your git config set up to recursively pull submodules, should just be a `git pull`? The submodules would be located in `app/context/static/portal-images`.

**Review by ilan-gold — APPROVED** [review]
Got it thanks!

**Comment by ilan-gold** [comment]
Oh no, just went to close out the tab @john-conroy and noticed the graph creeping up: 
Screen Shot 2021-04-20 at 12 08 33 PM

**Comment by ilan-gold** [comment]
It seems like the graph moves on every resize.

**Comment by john-conroy** [comment]
Thanks! Yeah don't worry about that for now. I'm going to fix overall layout later.

**Pull request merged**